src/mame/includes/ladybug.h

```diff
@@ -13,6 +13,7 @@
 #include "video/ladybug.h"
 #include "emupal.h"
 #include "tilemap.h"
+#include "machine/latch8.h"
```

Comment:
Please put this above the headers from src/emu (organise modules from most dependent to least dependent, to catch headers that don’t include dependencies faster).